lib/sqlalchemy/sql/selectable.py

```diff
@@ -5919,8 +5958,8 @@ def with_only_columns(
         _MemoizedSelectEntities._generate_for_statement(self)
 
         self._raw_columns = [
```

Comment:
**mike bayer** (zzzeek) wrote:

remove both

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325